downstream/modules/platform/proc-update-aap-rpm.adoc

```diff
@@ -0,0 +1,131 @@
+[id="proc-update-aap-rpm"]
+
+== Update planning
+
+Before you begin the update process, review the following considerations to plan and prepare your {PlatformNameShort} deployment:
+
+* Even if you have a valid license from a previous version, you must provide your credentials or a subscription manifest upon upgrading to the latest version of {PlatformNameShort}. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html/access_management_and_authentication/assembly-gateway-licensing#proc-attaching-subscriptions[Attaching your {PlatformName} subscription].
+
+* Clustered upgrades require special attention to instance and instance groups before upgrading. Ensure you capture your inventory or instance group details before upgrading. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html-single/configuring_automation_execution/index#controller-clustering[Clustering].
+
+* If you are currently running {EDAcontroller}, it is recommended that you disable all rulebook activations before upgrading to ensure that only new activations run after the upgrade process has completed. This prevents possibilities of orphaned containers running activations from the previous version. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html-single/using_automation_decisions/index#eda-enable-rulebook-activations[Enabling and disabling rulebook activations].
+
+== Choosing and obtaining a {PlatformName}installer
+
+Choose the {PlatformName} installer you need based on your {RHEL} environment internet connectivity. Review the following scenarios and decide on which {PlatformNameShort} installer meets your needs.
+Note
+
+A valid Red Hat customer account is required to access {PlatformName} installer downloads on the Red Hat Customer Portal.
+
+== Installing with internet access
+
+*Tarball install*
+
+.Procedure
+
+. Navigate to the link:access.redhat.com/downloads/content/480/ver=2.5/rhel---9/2.5/x86_64/product-software[{PlatformName} download download] page.
```

Comment:
Two "download"s in the link title here. 
You could also use:
link:{PlatformDownloadUrl}[{PlatformNameShort} download]

Reply:
File no longer used. It has been removed.